<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabConnection">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <widget class="QGroupBox" name="groupBoxNetwork">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>208</width>
         <height>157</height>
        </rect>
       </property>
       <property name="title">
        <string>Network</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelNetworkAddress">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditNetworkAddress"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelNetworkPort">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QSpinBox" name="spinBoxNetworkPort"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBoxUSB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>175</y>
         <width>208</width>
         <height>157</height>
        </rect>
       </property>
       <property name="title">
        <string>USB port</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelUSBDeviceName">
            <property name="text">
             <string>Device name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditUSBDeviceName">
            <property name="text">
             <string>/dev/ttyUSB0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelUSBBusPort">
            <property name="text">
             <string>Client address</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QSpinBox" name="spinBoxBusPort">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabPickup">
      <attribute name="title">
       <string>Pickup</string>
      </attribute>
      <widget class="QGroupBox" name="groupBoxPickupNetwork">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>266</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>Network</string>
       </property>
       <widget class="QLabel" name="labelPickupAddress">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>30</y>
          <width>116</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Pickup address</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxPickupAddress">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>50</y>
          <width>63</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tabMisc">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <widget class="QGroupBox" name="groupBoxDisplay">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>133</width>
         <height>157</height>
        </rect>
       </property>
       <property name="title">
        <string>Display update</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Text</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Progressbar</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QSpinBox" name="spinBoxDisplayTextUpdateRate">
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>250</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QSpinBox" name="spinBoxDisplayProgressBarUpdateRate">
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>155</x>
         <y>10</y>
         <width>143</width>
         <height>217</height>
        </rect>
       </property>
       <property name="title">
        <string>Alarm</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>SWR</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxAlarmSWR">
            <property name="suffix">
             <string>:1</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Forward power</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QSpinBox" name="spinBoxAlarmFwdPower">
            <property name="suffix">
             <string>W</string>
            </property>
            <property name="maximum">
             <number>15000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>1500</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Relected power</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QSpinBox" name="spinBoxAlarmRefPower">
            <property name="suffix">
             <string>W</string>
            </property>
            <property name="maximum">
             <number>1500</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>150</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
